runtime-config-linux.md

```diff
@@ -146,45 +146,128 @@ The cgroups will be created if they don't exist.
 
 `cgroupsPath` can be used to either control the cgroups hierarchy for containers or to run a new process in an existing container.
 
-Optionally, cgroups limits can be specified via `resources`.
+You can configure a container's cgroups via the `resources` field of the Linux configuration.
+Do not specify `resources` unless limits have to be updated.
+For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+
+#### Disable out-of-memory killer
 
 ```json
-    "resources": {
-        "disableOOMKiller": false,
-        "memory": {
-            "limit": 0,
-            "reservation": 0,
-            "swap": 0,
-            "kernel": 0,
-            "swappiness": -1
-        },
-        "cpu": {
-            "shares": 0,
-            "quota": 0,
-            "period": 0,
-            "realtimeRuntime": 0,
-            "realtimePeriod": 0,
-            "cpus": "",
-            "mems": ""
-        },
-        "blockIO": {
-            "blkioWeight": 0,
-            "blkioWeightDevice": "",
-            "blkioThrottleReadBpsDevice": "",
-            "blkioThrottleWriteBpsDevice": "",
-            "blkioThrottleReadIopsDevice": "",
-            "blkioThrottleWriteIopsDevice": ""
-        },
-        "hugepageLimits": null,
-        "network": {
-            "classId": "",
-            "priorities": null
-        }
-    }
+	"disableOOMKiller": false
 ```
 
-Do not specify `resources` unless limits have to be updated.
-For example, to run a new process in an existing container without updating limits, `resources` need not be specified.
+#### Memory
+
+```json
+	"memory": {
+		"limit": 0,
+		"reservation": 0,
+		"swap": 0,
+		"kernel": 0,
+		"swappiness": -1
+	}
+```
+
+#### CPU
+
+```json
+	"cpu": {
+		"shares": 0,
+		"quota": 0,
+		"period": 0,
+		"realtimeRuntime": 0,
+		"realtimePeriod": 0,
+		"cpus": "",
+		"mems": ""
+	}
+```
+
+#### Block IO Controller
+
+`blockIO` represents the cgroup subsystem `blkio` which implements the block io controller.
+For more information, see the [kernel cgroups documentation about `blkio`](https://www.kernel.org/doc/Documentation/cgroups/blkio-controller.txt).
+
+```json
+	"blockIO": {
+		"blkioWeight": 0,
+		"blkioLeafWeight": 0,
+		"blkioWeightDevice": [
+			{
+				"major": 8,
+				"minor": 0,
+				"weight": 500,
+				"leafWeight": 300
+			},
+			{
+				"major": 8,
+				"minor": 16,
+				"weight": 500
+			}
+		],
+		"blkioThrottleReadBpsDevice": [
+			{
+				"major": 8,
+				"minor": 0,
+				"rate": 600
+			}
+		],
+		"blkioThrottleWriteBpsDevice": null,
+		"blkioThrottleReadIOPSDevice": null,
```

Comment:
I'd  rather leave these out of the example instead of explicitly declaring them `null`.  You document them in the list of available attributes, and give other examples of the rate-setting structure.